wasm/install_deps.sh

```diff
@@ -2,5 +2,5 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 git clone https://github.com/emscripten-core/emsdk.git ${DIR}/emsdk
-${DIR}/emsdk/emsdk install latest
-${DIR}/emsdk/emsdk activate latest
+${DIR}/emsdk/emsdk install sdk-releases-upstream-edabe25af34554d19c046078f853999b074259ca-64bit
```

Comment:
https://emscripten.org/docs/getting_started/downloads.html#emsdk-install-targets

What about this?
```suggestion
${DIR}/emsdk/emsdk install 3.1.9
```

Or do I missmatch something?

Reply:
Already tried but it doesn't work with this tag :/

Reply:
After discussing this on slack, it turns out I was using an older version locally which `3.1.9` was not available. I will patch this